docs/DafnyRef/Statements.md

```diff
@@ -649,6 +649,8 @@ function usesTuple() : int
 }
 ```
 
+The assignment with failure operator `:-` returns (from the method) if the value evaluates to a failure value of a failure-compatible type, see [Section 19.7](sec-update-failure).
```

Comment:
I think without the parenthesis around 'from the method' would be better. AFAIK parenthesis are used to switch topics, but here the part 'from the method' doesn't do that.